doc/contents.rst

```diff
@@ -32,7 +32,15 @@ Sphinx documentation contents
 Indices and tables
 ==================
 
-* :ref:`genindex`
-* :ref:`modindex`
-* :ref:`search`
-* :ref:`glossary`
+.. only:: not epub and not epub3
```

Comment:
latex, text and some builders would not use these references.
We need to update the condition.